Source/ASVideoNode.mm

```diff
@@ -765,9 +778,11 @@ - (ASDisplayNode *)playerNode
 
 - (void)setPlayerNode:(ASDisplayNode *)playerNode
 {
-  ASDN::MutexLocker l(__instanceLock__);
-  _playerNode = playerNode;
-    
+  {
+    ASDN::MutexLocker l(__instanceLock__);
+    _playerNode = playerNode;
+  }
```

Comment:
This code path is so simple that `lock()` and `unlock()` makes sense: avoid creating/destroying a locker and save 1 line.